packages/account/src/Configs/trading-assessment-config.js

```diff
@@ -0,0 +1,305 @@
+import { generateValidationFunction, getDefaultFields } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+
+export const trading_assessment_questions = [
+    {
+        question_text: 'Do you understand that you could potentially lose 100% of the money you use to trade?',
+        section: 'risk_tolerance',
+        answer_options: [
+            { value: 'Yes', text: localize('Yes') },
+            { value: 'No', text: localize('No') },
+        ],
+        form_control: 'risk_tolerance',
+        field_type: 'radio',
+    },
+    {
+        question_text: 'How much knowledge and experience do you have in relation to online trading?',
+        section: 'source_of_experience',
+        form_control: 'source_of_experience',
+        answer_options: [
+            {
+                value: 'I have an academic degree, professional certification, and/or work experience.',
```

Comment:
Does the value need to be a whole sentence? Can we just a use some kind of a key instead?

Reply:
No.. The value that is to be sent has to be the whole sentence. The value is part of an enum that BE is checking for